sql/moz-fx-data-shared-prod/braze_derived/user_profiles_v1/checks.sql

```diff
@@ -1,10 +0,0 @@
--- macro checks
-
-#fail
-{{ not_null(["external_id", "email", "email_subscribe", "has_fxa", "create_timestamp", "update_timestamp"]) }}
-
-#fail
-{{ min_row_count(75000000) }}
-
-#fail
-{{ is_unique(["external_id", "email", "fxa_id_sha256"]) }}
```

Comment:
Should these provide a unique value as a set or should each of these columns contain unique values irrespectively of the other ones?

Reply:
@LiamMcFall I wonder if it would make sense to just remove email from the is_unique list in the bigConfig and leave it at that since we expect the same email to potentially appear more than once?